src/config.rs

```diff
@@ -787,7 +785,7 @@ pub struct CommandLineArgs {
     /// Tracing filter directives.
     ///
     /// See: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/struct.EnvFilter.html
-    #[clap(long, default_value = "info")]
+    #[clap(long, default_value = "warn")]
```

Comment:
Why was the default log level changed

Reply:
IMO warn should have been their in the first place, that's why I have approved it, but still curious about the explanation.

Reply:
notify-rust was a bit talkative so I thought it was better.

Reply:
Understand. Makes sense to me:)